src/Export.php

```diff
@@ -6,18 +6,37 @@
 use Illuminate\Http\Response;
 use Illuminate\Support\Arr;
 use Illuminate\Support\Collection;
+use League\Csv\CannotInsertRecord;
+use League\Csv\Exception;
 use League\Csv\Writer;
 use Statamic\Actions\Action;
 
 class Export extends Action
 {
+    /**
+     * Exclude statamic fields
+     */
+    const EXCLUDED_COLUMNS = [
+        'blueprint',
+        'updated_by',
+    ];
+
+    /**
+     * Title
+     */
+    public static function title()
+    {
+        return __('CSV Export');
+    }
+
     /**
      * Download items as CSV.
      *
      * @param $items
      * @param $values
      * @return false|Response
-     * @throws \League\Csv\CannotInsertRecord
+     * @throws CannotInsertRecord
+     * @throws Exception
```

Comment:
Can it throw both?

Reply:
I've changed the response from `$csv->getContent()` to `$csv->toString()`, and `$csv->toString()` @throws Exception If the string representation can not be returned